pytest_selenium/driver.py

```diff
@@ -38,31 +39,43 @@ def browserstack_driver(item, capabilities):
 
 def chrome_driver(item, capabilities):
     """Return a WebDriver using a Chrome instance"""
-    return webdriver.Chrome(desired_capabilities=capabilities)
+    options = item.config.option
+    kwargs = {'desired_capabilities': capabilities}
+    if options.driver_path is not None:
+        kwargs['executable_path'] = options.driver_path
+    return webdriver.Chrome(**kwargs)
 
 
 def firefox_driver(item, capabilities):
     """Return a WebDriver using a Firefox instance"""
     options = item.config.option
-    binary = None
-    if options.firefox_path:
+    kwargs = {'capabilities': capabilities or None}
+    if options.driver_path is not None:
+        kwargs['executable_path'] = options.driver_path
+    if options.firefox_path is not None:
         # get firefox binary from options until there's capabilities support
-        binary = FirefoxBinary(options.firefox_path)
-    profile = _create_firefox_profile(options)
-    return webdriver.Firefox(
-        firefox_binary=binary,
-        firefox_profile=profile,
-        capabilities=capabilities or None)
+        kwargs['firefox_binary'] = FirefoxBinary(options.firefox_path)
+    kwargs['firefox_profile'] = _create_firefox_profile(options)
+    return webdriver.Firefox(**kwargs)
 
 
 def ie_driver(item, capabilities):
     """Return a WebDriver using an Internet Explorer instance"""
-    return webdriver.Ie(capabilities=capabilities)
+    options = item.config.option
+    kwargs = {'capabilities': capabilities or None}
+    if options.driver_path is not None:
+        kwargs['executable_path'] = options.driver_path
+    return webdriver.Ie(**kwargs)
 
 
 def phantomjs_driver(item, capabilities):
     """Return a WebDriver using a PhantomJS instance"""
-    return webdriver.PhantomJS(desired_capabilities=capabilities)
+    options = item.config.option
+    _capabilities = capabilities or DesiredCapabilities.PHANTOMJS
```

Comment:
Where did `DesiredCapabilities.PHANTOMJS` come from? Is that something that was missing from the previous version but should have been there? Could you address this by making the default value of the `capabilities` argument `DesiredCapabilities.PHANTOMJS`, rather than having to use this `_capabilities` intermediary variable?